apps/wing-console/console/app/scripts/preview.mjs

```diff
@@ -1,13 +1,23 @@
 import { fileURLToPath } from "node:url";
+import { parseArgs } from "node:util";
 
 import open from "open";
 
 import { createConsoleApp } from "../dist/index.js";
 
+const options = parseArgs({
+  options: {
+    wingfile: {
+      type: "string",
+    },
+  },
+});
+
 const { port } = await createConsoleApp({
-  wingfile: fileURLToPath(
-    new URL("../../desktop/demo/index.w", import.meta.url),
-  ),
+  wingfile:
+    options.values.wingfile ??
+    fileURLToPath(new URL("../demo/index.w", import.meta.url)),
+  requestedPort: 3000,
```

Comment:
Port 3000 is usually used by docker. Perhaps it's better to use a different number for default

Reply:
https://www.gematrix.org/?word=wing+console

Wing Console in [Hebrew Gematria](https://www.gematrix.org/?word=wing+console#Hebrew-results) equals 1214